modules/apps/object/object-test/src/testFunctional/tests/paths/ObjectAdmin.path

```diff
@@ -15,6 +15,21 @@
 		<td>//li[@class='nav-item navbar-breakpoint-d-none']/../li[(@class='nav-item')]</td>
 		<td></td>
 	</tr>
+	<tr>
+		<td>SELECT_CUSTOM_OBJECT</td>
+		<td>//div[(@class='table-list-title') and contains (.,'${key_name}')]/../..//div[(@class='dnd-td item-actions')]</td>
+		<td></td>
+	</tr>
+	<tr>
+		<td>EDIT_CUSTOM_OBJECT</td>
+		<td>//a[(@class='dropdown-item')]</td>
+		<td></td>
+	</tr>
+	<tr>
+		<td>PUBLISH_CUSTOM_OBJECT</td>
+		<td>//button[@id='_com_liferay_object_web_internal_object_definitions_portlet_ObjectDefinitionsPortlet_publish']</td>
+		<td></td>
+	</tr>
```

Comment:
Please, remove it for now since we are not going to need it.

Reply:
@rodrigocunhaa  Done!